common/src/main/java/com/graphicsfuzz/common/util/FragmentShaderJobToVertexShaderJob.java

```diff
@@ -77,6 +81,27 @@ public static ShaderJob convertShaderJob(ShaderJob sj) throws Exception {
     TranslationUnit vertexShader = sj.getFragmentShader().get();
     vertexShader.setShaderKind(ShaderKind.VERTEX);
 
+    // Analyse the shader for used features (can't use anonymous class due to member access)
+    class Analysis extends StandardVisitor {
+      public boolean usesFragCoord = false;
+      public boolean usesFragDepth = false;
+
+      @Override
+      public void visitVariableIdentifierExpr(VariableIdentifierExpr variableIdentifierExpr) {
+        super.visitVariableIdentifierExpr(variableIdentifierExpr);
+        if (variableIdentifierExpr.getName().equals(OpenGlConstants.GL_FRAG_COORD)) {
+          usesFragCoord = true;
+        }
+        if (variableIdentifierExpr.getName().equals(OpenGlConstants.GL_FRAG_DEPTH)) {
+          usesFragDepth = true;
+        }
+      }
+    }
+    
+    final Analysis analysis = new Analysis();
```

Comment:
Rename variable in accordance with how you rename the class.

Reply:
done